internal/config/config.go

```diff
@@ -47,9 +47,10 @@ type IndexerConfig struct {
 }
 
 type SubmitConfig struct {
-	Address    string `yaml:"address"      envconfig:"SUBMIT_TCP_ADDRESS"`
-	SocketPath string `yaml:"socketPath"   envconfig:"SUBMIT_SOCKET_PATH"`
-	Url        string `yaml:"url"          envconfig:"SUBMIT_URL"`
+	Address             string `yaml:"address"      envconfig:"SUBMIT_TCP_ADDRESS"`
+	SocketPath          string `yaml:"socketPath"   envconfig:"SUBMIT_SOCKET_PATH"`
+	Url                 string `yaml:"url"          envconfig:"SUBMIT_URL"`
+	BlockFrostProjectID string `yaml:"blockFrostProjectID"` envconfig:"SUBMIT_BLOCKFROST_PROJECT_ID"`
```

Comment:
There's an extra ` in your code.
```suggestion
	BlockFrostProjectID string `yaml:"blockFrostProjectID" envconfig:"SUBMIT_BLOCKFROST_PROJECT_ID"`
```

Reply:
Fixed, thanks